tools/clang/test/CodeGenSPIRV/binary-op.mul-assign.type-mismatch.hlsl

```diff
@@ -1,53 +0,0 @@
-// RUN: %dxc -T ps_6_2 -E main -fspv-target-env=vulkan1.1 -enable-16bit-types
```

Comment:
Can you use `git mv` for these so that the history is preserved correctly?

Reply:
Tried modify the file in one commit first, then git mv in another commit.

Will this preserve history after squash?

Reply:
Hopefully!